presto-verifier/src/main/java/com/facebook/presto/verifier/PrestoVerifier.java

```diff
@@ -113,6 +116,9 @@ public int run(String[] args)
             queries = applyOverrides(config, queries);
             queries = filterQueryTypes(new SqlParser(getParserOptions()), config, queries);
             queries = filterQueries(queries);
+            if (config.getShadowWrites()) {
```

Comment:
Shouldn't this option imply that write queries are allowed. As it stands, it's possible to say only READ queries are allowed, but to enable write shadowing. I find this confusing.